librz/sign/create.c

```diff
@@ -60,6 +60,17 @@ static RzFlirtFunction *flirt_function_new(const char *name, bool is_local, ut64
 	return function;
 }
 
+static ut32 flirt_crc16_length(RZ_NONNULL const ut8 *mask, size_t size) {
```

Comment:
please add a comment explaining why we compute the flirtcrc length the way we do.